bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -38,28 +44,63 @@
     private boolean disableHvac = false;
 
     private @Nullable Double batteryLevel;
+    private @Nullable Double batteryAvailableEnergy;
+    private @Nullable Integer chargingRemainingTime;
+    private @Nullable String chargingStatus;
+    private @Nullable String chargingMode;
     private @Nullable Boolean hvacstatus;
+    private Double hvacTargetTemperature = 20.0;
```

Comment:
As far as I understand this value can not be got from the car it is just a pre-heat target temperature that can have the value 19, 20 or 21 on my car.  This seams to also be the feedback from users and their research in the python library forums.

I did discuss scrapping it and hard coding 20.  But it might work on some cars and in the future.  So we decided to include it and set it to be "Advanced".  The sitemap restricts it to 19, 20, 21 values.  So it is there but should not be messed with and set to 20.  This hopefully explains it in the code.

double !Double is a good idea.